RELEASE/scripts/autoscend/combat/auto_combat_util.ash

```diff
@@ -354,6 +361,21 @@ skill getStunner(monster enemy)
 			return $skill[Blood Chains];
 		}
 		break;
+	case $class[Pig Skinner]:
+	case $class[Cheese Wizard]:
+	case $class[Jazz Agent]:
+		if(canUse($skill[Noogie]))
+		{
+			return $skill[Noogie];
+		}
+		if(canUse($skill[Gather Cheese-Chi]) && my_hp() <= my_maxhp() - 30)
```

Comment:
I added the health conditional because Gather Cheese-Chi is a spammable heal and I didn't understand autoscend combat at the time I wrote this. Because of Emmental Elemental, I am removing the health conditional and just use it as a Stun that happens to heal, not a Heal that happens to stun.